udtube/data/conllu.py

```diff
@@ -20,50 +20,89 @@ class Error(Exception):
 class ID:
     """Representation of a CoNLL-U sentence ID.
 
-    Most of the time this is just a single integer, but MWEs use two integers
-    to represent a span of tokens.
+    Most of the time this is just a single integer. Two other possibilities
+    exist, however:
+
+    * decimals: represented by an integer and a second integer > 0.
+    * MWEs: represented by two integers denoting a span of tokens.
+
+    The decimal dummy value is 0; the MWE dummy value (for the upper element
+    of the span) is the same as the lower value.
```

Comment:
I'd add somewhere that MWEs and decimals cannot co-occur (based on the slice code, I am assuming this is the case)

Reply:
Done